analyzers/rspec/cs/S2222_c#.html

```diff
@@ -1,7 +1,9 @@
-<p>If a lock is acquired and released within a method, then it must be released along all execution paths of that method.</p>
+<p>If a lock is known to be held or acquired, and then released within a method, then it must be released along all execution paths of that
+method.</p>
 <p>Failing to do so will expose the conditional locking logic to the method’s callers and hence be deadlock-prone.</p>
-<p>The types tracked by the rule are: <code>Monitor</code>, <code>Mutex</code>, <code>ReaderWriterLock</code>, <code>ReaderWriterLockSlim</code> and
```

Comment:
Actually it should be <p>The types tracked by the rule are: <code>Monitor</code>, <code>Mutex</code>, <code>ReaderWriterLock</code>, <code>ReaderWriterLockSlim</code> and
<code>SpinLock</code> from the <code>System.Threading</code> namespace.</p>
So please revert.

Reply:
That doesn't represent current `master` state of the `rspec` repository.

The change will get reintroduced in the S2222 SonarWay, because it sits on that branch in `rspec` repo

Reply:
I might be missing something, but what does S2222 being in SonarWay or not has to do with the fact that there are <code> tags missing in the rspec?

Reply:
Understood. I'll approve this PR, we can merge it. However if S2222 does not go into SonarWay for this release, we should do a separate PR on RSPEC to fix the quotes and update the RSPECs one more.